src/playwright/transformPlaywrightJson.ts

```diff
@@ -120,7 +120,7 @@ export const transformPlaywrightJson = (index: V3StoriesIndex | V4Index | Unsupp
       Object.values((index as V3StoriesIndex).stories)
     );
     titleIdToEntries = v3TitleMapToV4TitleMap(titleIdToStories);
-  } else if (index.v === 4) {
+  } else if (index.v === 4 || index.v === 5) {
```

Comment:
```suggestion
  // v4 and v5 are pretty much similar, so we process it in the same way
  } else if (index.v === 4 || index.v === 5) {
```

Reply:
In my opinion, we should support higher versions per default and react to issues reactively instead of adjusting the index every time we bump it.